vanagon.gemspec

```diff
@@ -7,13 +7,14 @@ Gem::Specification.new do |gem|
 
   gem.summary = "Another mega-package build tool, now with more Make"
   gem.description = "Vanagon is a tool to build a single package out of a project, which can itself contain one or more components."
+  gem.license = "Apache-2.0"
 
   gem.authors  = ['Puppet Labs']
   gem.email    = 'info@puppetlabs.com'
   gem.homepage = 'http://github.com/puppetlabs/vanagon'
 
   gem.add_development_dependency('rspec', ["~> 3.0"])
-  gem.add_development_dependency('yard')
+  gem.add_development_dependency('yard', '~> 0.8')
```

Comment:
yard generation also requires redcarpet, but for some reason yard doesn't pull it in.